crates/bevy_render/src/camera/camera.rs

```diff
@@ -193,8 +193,8 @@ impl Camera {
 
     /// Given a position in world space, use the camera to compute the viewport-space coordinates.
     ///
-    /// To get the coordinates in Normalized Device Coordinates, you should use
-    /// [`world_to_ndc`](Self::world_to_ndc).
+    /// To get the coordinates in Normalized Device Coordinates,
+    /// you should use [`world_to_ndc`](Self::world_to_ndc).
```

Comment:
please don't make unrelated doc uglier by introducing illogical line breaks

Reply:
Since there was already an awkward line-break, I personally found it easier to read when the sentence is split at the comma.
Happy to revert if ya don't like it :)

Reply:
I like it better this way but we should revert to reduce churn :p

Reply:
having full lines make it easier to read, rather than cutting just in the middle of the line. current line break is the logical one to have the most complete line

but also, changing things unrelated in a PR make it harder to track something origin